spec/requests/api/members_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe "api/members" do
+
+  def json
+    JSON.parse(response.body)
+  end
+
+  describe 'POST api/members' do
+    let(:new_member) { build :member, email: "newbie@test.org" }
```

Comment:
I wanted to keep it in case I'll want to write more specs (this is very much WIP). I think I can only reliably get an email address from the home page signup form, but I'll add more fields, should we have a signup form that contains more data.